addons/website/static/lib/multirange/multirange_custom.js

```diff
@@ -295,11 +295,18 @@ class Multirange {
     }
 
     formatNumber(number) {
-        number = String(number).split('.');
-        if (number[1] && number[1].length === 1) {
-            number[1] += '0';
+        const locale = document.querySelector("html").getAttribute("lang");
```

Comment:
Relying on the `lang` attribute means that you can't use multirange anywhere, as it's not set in the backend, for example. I'm wondering what should be the preferred method for retrieving the locale; it seems that we are supposed to retrieve it from the user service, but that requires the user service to be accessible and initialised, which is not always easily feasible.

If there is no context other than the website where multirange input is used, I'm fine with the current limitation though. Tell me if this sounds ok to you as well 👍️

Reply:
Yes you're right, my change makes this widget very linked to the `website_*` modules. But since it's defined in `website`, I'm not too worried. If someone wants to use it one day in the backend, I imagine they'll put it in `web` and probably have to adapt it. I'm not sure it's worth adapting this component in case one day someone wants to use it in the backend..